packages-ts/gauntlet-terra-contracts/src/commands/contracts/proxy_ocr2/deploy.ts

```diff
@@ -13,8 +14,12 @@ type ContractInput = {
 }
 
 const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInput> => {
+  const rdd = getRDD(flags.rdd)
+  const contract = args[0]
```

Comment:
Thanks @momentmaker ! Could you add it to propose and accept proxy commands as well?

Reply:
@RodrigoAD sure done :)